Libraries/LibTTF/Font.h

```diff
@@ -100,7 +100,7 @@ class Font : public RefCounted<Font> {
     }
 
     // This owns the font data
-    ByteBuffer m_buffer;
+    ReadonlyBytes m_buffer;
```

Comment:
Oh wait back up. `ReadonlyBytes` is non-owning, so this doesn't seem safe.
I didn't realize that `Font` would take ownership of the `ByteBuffer`.